blog/_posts/2019-01-18-fluxdiffeq.md

```diff
@@ -10,12 +10,13 @@ robustly add differential equation solvers to neural networks in Julia.
 ![Flux ODE Training Animation](https://user-images.githubusercontent.com/1814174/51399500-1f4dd080-1b14-11e9-8c9d-144f93b6eac2.gif)
 
 The [Neural Ordinary Differential Equations](https://arxiv.org/abs/1806.07366)
-paper has been a hot topic even before it made a splash as Best Paper of NeurIPS
-2018. Among other demonstrations, combining these two disparate fields enabled
-the authors to reduce the number of parameters and reduce memory costs in
-comparison to ResNet. But this is only the beginning: neural networks and
-differential equations were born to be together. This blog post will explain
-why, and start to give a sense of what's possible with state-of-the-art tools.
+paper, by Jesse Bettencourt and collaborators, has been a hot topic even before it

```

Comment:
No, we can't do this.
It is against academic convention to refer to the paper as _by 3rd author and collaborators_ . 

The paper is listed as by
Ricky T. Q. Chen, Yulia Rubanova, Jesse Bettencourt, David Duvenaud

we thus need to say either

by _Ricky T. Q. Chen and collaborators_

or 

_by Ricky T. Q. Chen, Yulia Rubanova, Jesse Bettencourt and David Duvenaud_

Reply:
This is not actually a paper, so it's less important for us to follow that kind of convention unless it's likely to cause frustration for other reasons. I'll try and reword it though.